src/main/java/seedu/address/logic/commands/UndoCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Deletes a person identified using it's displayed index from the address book.
+ */
+public class UndoCommand extends Command {
+
+    public static final String COMMAND_WORD = "undo";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Erases the previous change to the address book and restores the the older state.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_UNDO_SUCCESS = "Undid last change.";
+
```

Comment:
Can add another message if the undo failed? For example just launching the app and using undo immediately.:grin: